README.md

```diff
@@ -281,17 +290,17 @@ A. Probably because the database is being queried every time and this uses a
    unless it detects that the database has changed since the file was created,
    in which case it will still use the database to ensure accuracy.
 
-Q. bash-completion interferes with my `command_not_found_handler` function!
+**Q. bash-completion interferes with my `command_not_found_handler` function!**
 
 A. If your `command_not_found_handler` function is not intended to
    address (possibly missing) commands invoked during bash
-   programmable completion functions, you can take that into account
-   in your function for example by testing if the `$COMP_`* variables
+   programmable completion functions. You can account for this
```

Comment:
Replacing the comma with the period and splitting the sentence like that (before "You") seems clunky here. Was this intentional? Or did you also intend to remove the "If" from the start of the sentence?